source/_components/livebox.markdown

```diff
@@ -0,0 +1,101 @@
+---
+title: "Livebox"
+description: "Instructions on how to integrate Orange Livebox routers into Home Assistant."
+logo: orange_livebox.svg
+ha_category:
+  - Network
+  - Presence Detection
+  - Sensor
+ha_release: 0.99
+ha_iot_class: Local Polling
+---
+
+The `livebox` integration allows you to observe and control [Livebox router](http://www.orange.fr/).
+
+There is currently support for the following device types within Home Assistant:
+
+* [Sensor](#sensor) with traffic metrics
+* [Device tracker](#presence-detection) for connected devices
+
+## Configuration
+
+The preferred way to setup the Orange Livebox platform is by enabling the discovery component.
+Add your equipment via the Integration menu
+
+If you have enabled the [discovery component](/components/discovery/),
+your Livebox should be detected automatically. Otherwise, you can set it
+up manually in your `configuration.yaml` file:
+
+```yaml
+livebox:
```

Comment:
We can remove `port` and `username` from this example, because we want only required items in the first example for the simple copy and paste work. But you can add an advanced example with the optional config variables included.